our-work/collaborations/nlp.yml

```diff
@@ -0,0 +1,18 @@
+title: NLP Syllabi Project 
+id: nlp-syllabi
+description: The Brown University Library Staff were interested in investigating how race and racial issues are discussed in courses at Brown. 
+tags:
+  - natural language processing
```

Comment:
Both software of collaborations have a schema that is issued to validate the tags. If you have tags that already exist in there, reuse. If the tag is nw, you'll need to add to the schema. We've been using lower case and dashes instead of spaces.